lib/trailblazer/rails/railtie/loader.rb

```diff
@@ -1,56 +1,52 @@
+gem 'trailblazer-loader'
```

Comment:
do you need this too?

Reply:
We need the gem to show a error that the gem is missing in case the dev forget to add it.